inc/ad-servers/class-ad-layers-dfp.php

```diff
@@ -534,8 +534,14 @@ private function ad_unit_js( $ad_layer ) {
 					}
 					$sizes = apply_filters( 'ad_layers_dfp_ad_unit_sizes', $sizes, $ad_unit, $breakpoint );
 
+					// If we have no defaults, assume to use all of the sizes for this breakpoint.
+					if ( empty( $this->default_by_unit[ $ad_unit['code'] ] ) ) {
+						$this->default_by_unit[ $ad_unit['code'] ] = $sizes;
+					}
+
 					// Generate the mapping JS and store it with the unit.
-					$unit_key = $this->sanitize_key( $ad_unit['code'] );
+					// TODO: streamline targeting so mappings and units are 1:1.
+					$unit_key = $ad_unit['code'];
```

Comment:
is removing the call to sanitize_key going to cause issues? We may end up with some weird characters in the key, and it removes a call to a filter that people may be using.

Reply:
Yes, I never updated that to use `sanitize_title_with_dashes` or remove the todo. Thanks for the reminder.